src/app/components/feed/CreateFeed.js

```diff
@@ -0,0 +1,350 @@
+            defaultMessage="Memorable feed title"
+            description="Placeholder text for feed title field"
+          >
+            { placeholder => (
+              <TextField
+                id="create-feed__title"
+                placeholder={placeholder}
+                label={<FormattedMessage
+                  id="createFeed.titleLabel"
+                  defaultMessage="Title"
+                  description="Label for the shared feed title input"
+                />}
+                helpContent={<FormattedMessage
+                  id="createFeed.titleHelper"
+                  defaultMessage="Great shared feed names are short, memorable, and tell your audience the focus of the media"
+                  description="Title input helper text"
+                />}
+                value={title}
+                onChange={e => setTitle(e.target.value)}
+                required
+              />
+            )}
+          </FormattedMessage>
+          <FormattedMessage
+            id="createFeed.descriptionPlaceholder"
+            defaultMessage="Give this shared feed an optional description."
+            description="Placeholder text for feed description field"
+          >
+            { placeholder => (
+              <TextArea
+                id="create-feed__description"
+                placeholder={placeholder}
+                label={<FormattedMessage
+                  id="createFeed.descriptionLabel"
+                  defaultMessage="Description"
+                  description="Label for a field where the user inputs text for a description to a shared feed"
+                />}
+                value={description}
+                onChange={e => setDescription(e.target.value)}
+              />
+            )}
+          </FormattedMessage>
+        </div>
+        <div className={styles.createFeedCard}>
+          <div className="typography-subtitle2">
+            <FormattedMessage
+              id="createFeed.feedContentTitle"
+              defaultMessage="Feed content"
+              description="Title of section where a list can be selected as the content of the feed"
+            />
+          </div>
+          <FormattedMessage
+            id="createFeed.feedContentBlurb"
+            defaultMessage="Select a filtered list of fact-checks from your workspace to contribute to this shared feed. You will be able to update this list at any time."
+            description="Helper text for the feed content section"
+          />
+          <FormattedHTMLMessage
+            id="createFeed.feedContentBlurb2"
+            defaultMessage="<strong>Note:</strong> Your list must contain <strong>published fact-checks</strong> in order to be part of this shared feed."
+            description="Helper text for the feed content section"
+          />
+          <SelectListQueryRenderer
+            value={selectedListId}
+            onChange={e => setSelectedListId(e.target.value)}
```

Comment:
```suggestion
            onChange={e => setSelectedListId(+e.target.value)}
```
This wasn't saving for me locally until I coerced the value of the list ID to a `Number`